docs/designs/topology.md

```diff
@@ -186,128 +240,170 @@ spec:
       key: "topology.kubernetes.io/numa"
 ```
 
-**Creating ClusterTopology:**
+**Configuring ClusterTopology:**
+
+The Manged ClusterTopology CR is generated and managed sole by the Grove operator. To configure it:
 
-1. Customize example above with your cluster's actual `key` values
-2. Choose a name for your topology:
-   - Use custom name (e.g., "my-cluster-topology") OR
-   - Use default name "grove-topology" (no config needed)
-3. Create resource: `kubectl apply -f clustertopology.yaml`
-4. If using custom name: configure operator with topology name in operator config
-5. See Admin Responsibilities section for additional setup requirements
+1. Define topology levels in OperatorConfiguration under `clusterTopology.levels` (see OperatorConfiguration section below)
+2. Set `clusterTopology.enabled: true` in OperatorConfiguration
+3. Restart the Grove operator
+4. Operator creates ClusterTopology CR named "grove-topology"
+5. Operator creates and continuously reconciles KAI Topology CR
+6. If configuration is invalid or ClusterTopology CR creation fails → operator exits with error
+7. If downstream topology CR creation fails (KAI Topology) → reflected in ClusterTopology Ready condition (operator continues)
 
-**Validation:**
 
-- Level names must be from predefined set: region, zone, datacenter, block, rack, host, numa (enum validation)
-- Each level `domain` and `key` must be unique
-- Admins can skip intermediate levels (e.g., define only region, rack, host)
-- Partially immutable after creation (see Characteristics above for details)
-- Deletion protection via controller finalizer (blocks deletion while PodCliqueSet resources reference this topology OR
-  topology is enabled)
+**A. Webhook Validation**
 
-**Mutation Webhook:**
+The validation webhook enforces business logic constraints:
 
-The mutation webhook automatically modifies ClusterTopology resources on CREATE operations:
+- **On CREATE**: Validates domain uniqueness, key uniqueness, and key format
+- **On UPDATE**: Runs same CREATE validations (uniqueness, format)
+- **Authorization**: Only operator service account can CREATE/UPDATE/DELETE the Manged ClusterTopology
```

Comment:
```suggestion
- **Authorization**: Only operator service account can CREATE/UPDATE/DELETE the Manged ClusterTopology with the resourceName grove-topology
```

Reply:
Or actually, looks like we describe this below. Maybe this is just duplicate and we should remove it favour of the section below?